docs/source/conf.py

```diff
@@ -352,6 +352,7 @@ def get_incollection_template(self, e):
     "examples/zne-braket-ionq": "_static/zne-braket-ionq.png",
     "examples/bqskit": "_static/bqskit.png",
     "examples/simple-landscape-qiskit": "_static/simple-landscape-qiskit.png",
+    "examples/simple-landscape-pennylane": "_static/simple-landscape-pennylane.png",
```

Comment:
Validation is failing because the name is too long. Don't forget to update the filename also when applying this suggestion 🙂  
```suggestion
    "examples/simple-landscape-pennylane": "_static/simple-landscape-pln.png",
```